modules/dqt/jsx/react.navigationStepper.js

```diff
@@ -0,0 +1,172 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+
+import {Stepper} from './components/stepper';
+
+const NavigationStepper = (props) => {
+  const mySteps = ['Info', 'DefineFields', 'DefineFilters', 'ViewData'];
+
+  const [
+    // eslint-disable-next-line no-unused-vars
```

Comment:
We don't disable our static analysis tools just to get them to pass. If this isn't used, delete it.

Reply:
@driusan might be best solution unless eslint has something for not complaining about state in react function components.

Reply:
@driusan latest change reverts it.